client-templates/gitlab/accept.json.sample

```diff
@@ -118,7 +118,19 @@
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
-      "path": "/api/v4/projects/:project/repository/files*%2F*req*.txt",
+      "path": "/api/v4/projects/:project/repository/files*%2Frequirements/*.txt",
```

Comment:
Guess this change wasn't intentional since it was not done to any other SCM and also it's duplicated a few lines below